moto/sqs/responses.py

```diff
@@ -596,6 +600,12 @@ def list_queue_tags(self):
             <Value>{{ message.group_id }}</Value>
           </Attribute>
           {% endif %}
+          {% if message.aws_trace_header is not none %}
```

Comment:
This check needs to be modified as well. I'm guessing that's blocking the tests, so when they pass, I'll be happy to merge

Reply:
Done